provisioning/osc-install

```diff
@@ -248,25 +249,34 @@ do_post(){
   # Create router infrastructure
   CA=/etc/openshift/master
 
-  osadm create-server-cert --signer-cert=$CA/ca.crt \
+  oadm create-server-cert --signer-cert=$CA/ca.crt \
   --signer-key=$CA/ca.key --signer-serial=$CA/ca.serial.txt \
   --hostnames="*.${OPENSHIFT_BASE_DOMAIN}" \
   --cert=$CA/cloudapps.crt --key=$CA/cloudapps.key
 
   cat $CA/cloudapps.crt $CA/cloudapps.key $CA/ca.crt > $CA/cloudapps.router.pem
 
-  osadm router --default-cert=$CA/cloudapps.router.pem \
+  oadm router router-default --replicas=2 --default-cert=$CA/cloudapps.router.pem \
   --credentials=/etc/openshift/master/openshift-router.kubeconfig \
   --selector='region=infra' \
-  --images='registry.access.redhat.com/openshift3_beta/ose-${component}:${version}'
+  --images='registry.access.redhat.com/openshift3/ose-${component}:${version}'
 
   # Create registry
   mkdir -p /mnt/registry
-  osadm registry --create \
-  --credentials=/etc/openshift/master/openshift-registry.kubeconfig \
-  --images='registry.access.redhat.com/openshift3_beta/ose-${component}:${version}' \
-  --selector="region=infra" --mount-host=/mnt/registry
 
+  echo '{"kind":"ServiceAccount","apiVersion":"v1","metadata":{"name":"registry"}}' | oc create -f -
+  oc get scc privileged -o yaml > priv.yaml
+  if [ $(grep -c 'system:serviceaccount:default:registry' priv.yaml) -eq 0 ]; then
+    echo '- system:serviceaccount:default:registry' >> priv.yaml
+    oc update scc privileged -f priv.yaml
+  fi
+
+  oadm registry --service-account=registry \
+  --config=/etc/openshift/master/admin.kubeconfig \
+  --credentials=/etc/openshift/master/openshift-registry.kubeconfig \
+  --images='registry.access.redhat.com/openshift3/ose-${component}:${version}' \
+  --selector='region=infra' \
+  --mount-host=/mnt/registry
 }
 
 # Process input
```

Comment:
This may be for a follow-up PR, but should the "beta4" be updated on line 311 below?